.github/workflows/seqera_docs_changelog.yml

```diff
@@ -0,0 +1,60 @@
+name: Push changelog to Seqera Docs
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      release_name:
+        description: 'Release version (e.g. 1.0.0)'
```

Comment:
Can we validate the release version matches an expected release version format. 


```
      - name: Validate inputs
        run: |
          RELEASE_NAME="${{ github.event.release.name || inputs.release_name }}"
          if [[ ! $RELEASE_NAME =~ ^v[1-9]\.[0-9]{1,2}\.[0-9]{1,2}$ ]]; then
            echo "Error: Invalid release name format. Expected version between v1.0.0 and v9.99.99"
            exit 1
          fi
```

I appreciate that this regex does not meet the entire semver specification however it will cover our explicit usecase.

Side note this thread is a brilliant example of why it is so basic.  https://gist.github.com/jhorsman/62eeea161a13b80e39f5249281e17c39

Reply:
Why do we need to validate it?

I'm not sure that we should to be honest. By the time this action runs, the release is already minted. I think that the changelog should match the release name, whatever format that took.

Reply:
This is a fair point this also need to go through human review at the docs side.